vizualization/hq_event_wrapper.py

```diff
@@ -0,0 +1,60 @@
+import subprocess as sp
+import os
+import json
+from events_type import *
+
+
+class Events(list):
+    def __init__(self, data):
+        self.extend(data)
+
+    def get_by_type(self, event_type) -> list:
+        if type(event_type) not in [list, tuple]:
+            event_type = [event_type]
+        return [i for i in self if i['type'] in event_type]
+
+    def get_by_worker_id(self, id) -> list:
+        return [i for i in self.get_by_type(WorkerEvents) if i['id'] == id]
+
+
+class HQEventWrapper:
+
+    def __init__(self, hq_event_bin_path: str, cwd: str = "", hq_bin_path: str = ""):
```

Comment:
Let's define a function that will receive a path to a JSON file and return the parsed events. For now we can just generate the JSON from CLI.